packages/toolpad-app/src/components/HelpTooltipIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Tooltip, TooltipProps } from '@mui/material';
+import HelpOutlineIcon from '@mui/icons-material/HelpOutline';
+
+export interface HelpTooltipIconProps extends Omit<TooltipProps, 'title' | 'children'> {
+  helpText: string | React.ReactNode;
```

Comment:
```suggestion
  helpText: React.ReactNode;
```

Actually my bad, looks like `string |` is redundant here and `React.ReactNode` includes strings among other things. I didn't know until I checked a while ago!